source/standards/tracking-dependencies.html.md.erb

```diff
@@ -16,54 +16,59 @@ Our [programming language style guides][] also contain language-specific advice
 
 ## Update dependencies frequently
 
-Update your dependencies frequently rather than in ‘big bang’ batches. This works well with [continuous delivery][] principles, ensuring the changes introduced are small and can be automatically tested.
+Update your dependencies frequently rather than in ‘big bang’ batches. This works well with [continuous delivery][] principles and makes sure the changes introduced are small and can be automatically tested.
 
-Tools exist that will scan GitHub repositories and raise PRs when dependency updates are found. Teams at GDS are using:
+There are tools which scan GitHub repositories and raise PRs when they find dependency updates. Teams at GDS are using:
 
-* [Dependabot][] - used by GOV.UK, GOV.UK Pay and GovWifi. The GOV.UK team manual contains [guidance on using dependabot][] and [how the PRs raised should be reviewed][]
+* [Dependabot][] - used by GOV.UK, GOV.UK Pay and GovWifi. The GOV.UK team manual contains [guidance on using Dependabot][] and [how the PRs raised should be reviewed][]
```

Comment:
I think there's a couple more teams/ products using dependabot now.
Don't let this block this PR. I'll ask and come back either here or make an issue